### ISSUE TITLE
System reboots when pressing FN+F1 for sleep mode

### ISSUE THREAD

**Issue by systemfaliure** [systemfaliure/ASUS-X556UQK-hackintosh#3]

**Comment by wuyiwuyaya** [comment]
You can disable it

**Comment by systemfaliure** [comment]
Yes I know, still I would like to make it functional.

In data 22 ottobre 2018 17:22:42 wuyiwuyaya <notifications@github.com> ha 
scritto:

**Comment by wuyiwuyaya** [comment]
Will it restart during sleep?

**Comment by systemfaliure** [comment]
Exactly. After pressing FN+F1 the screen turns off and after a while the 
computer just restarts.

**Comment by sav-valerio** [comment]
If I click on the sleep command in the Apple menu it works just fine. System goes properly in sleep mode (even if it's slower than on Windows) 

If I try to put the system in sleep using instead FN+F1 keys the computer just restarts.

**Comment by systemfaliure** [comment]
If I click on the sleep command in the Apple menu it works just fine. 
System goes properly in sleep mode (even if it's slower than on Windows)

If I try to put the system in sleep using instead FN+F1 keys the computer 
just restarts.

**Comment by wuyiwuyaya** [comment]
My computer will restart during sleep and can't sleep for a long time. Do you know why? It’s useless to use your patch.

**Comment by systemfaliure** [comment]
It seems fine on my machine, no problems with few hours of sleep.

I will try to put the computer in sleep mode for the whole night and report 
back.

**Comment by systemfaliure** [comment]
**Update:** My system stayed more than 9 hours in sleep mode. 
It then woke up after my command without any problem, with everything opened as before.

You should check your configuration/kexts for any conflict with sleep mode.
Also make sure that you have disabled hibernation with the following commands in the Terminal:

```
sudo pmset -a hibernatemode 0
sudo rm /var/vm/sleepimage

```
